source/code/plugins/patch_management_lib.rb

```diff
@@ -158,6 +158,7 @@ def self.availableUpdatesXMLtoHash(availableUpdatesXML, os_short_name)
                                 OMS::Common.format_time(availableUpdatesHash["BuildDate"].to_i)
         ret["Repository"] = availableUpdatesHash.key?("Repository") ? availableUpdatesHash["Repository"] : nil
         ret["Installed"] = false
+        ret["UpdateState"] = "Needed"
```

Comment:
Since we only send Available updates, we might retire "Installed" field in future. Right now ingestion is referring to "Installed", I am switching it to use "UpdateState" instead.